surveys/tests/test_urls.py

```diff
@@ -15,18 +15,20 @@ def test_survey_form(self):
         )
 
     def test_survey_post(self):
+        user_id = 'User#20@!_ Id'
+        quoted_user_id = 'User%2320@!_%20Id'
```

Comment:
These can go on the class (`self.user_id`, `self.quoted_user_id`) to save on some duplication if we're using them twice. :)